README.md

```diff
@@ -47,3 +47,6 @@ Configure github action at https://docs.github.com/en/pages/getting-started-with
 3. push pages to github. A build action will be triggered automatically to release the page https://github.com/hippopenny/hippopenny.github.io/actions
 
 
+
+
+
```

Comment:
can you please add a section with instruction on how to generate pages?

Reply:
I will add instructions on how to create it manually (previously I planned to write it in the cron section).